init-environment.sh

```diff
@@ -0,0 +1,25 @@
+# Initializes environment variables necessary for running
+# tests against docker containers.
+
+alias echoerr=">&2 echo"
```

Comment:
An `echo` inside of a function will be hidden because stdout is used for piping output from a function by default. Have to log to stderr to print debugging info related to failure inside of a function.